README.md

```diff
@@ -102,7 +102,7 @@ Please make sure you use Python 2.x. Python 3.x is not supported currently.
 To build for macOS, you need [Xcode 12 beta 3](https://developer.apple.com/xcode/resources/).
 The required version of Xcode changes frequently, and is often a beta release.
 Check this document or the host information on <https://ci.swift.org> for the
-current required version.
+current required version. The path name for the Xcode app must not include any spaces.
```

Comment:
Sounds like we should tweak to wording to make it clear we’re flagging an existing bug you might hit (can we get a Jira for it too rather than just a radar?). Do we know yet if the bug is in the script vs in some dependency that makes it harder to fix?

Reply:
So something like "Due to [link to jira] the Xcode app must not include any spaces.

Reply:
@airspeedswift Ben, I took a crack at fixing it.